cloudfoundry/resource_cf_route_service_binding.go

```diff
@@ -105,6 +138,34 @@ func resourceRouteServiceBindingDelete(ctx context.Context, d *schema.ResourceDa
 
 	serviceID := d.Get("service_instance").(string)
 	routeID := d.Get("route").(string)
-	_, err := session.ClientV2.DeleteServiceBindingRoute(serviceID, routeID)
+
+	var err error
+
+	options := client.NewServiceRouteBindingListOptions()
+	options.ServiceInstanceGUIDs = client.Filter{Values: []string{serviceID}}
+	options.RouteGUIDs = client.Filter{Values: []string{routeID}}
+	routeBindings, err := session.ClientGo.ServiceRouteBindings.ListAll(context.Background(), options)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	routeBindingID := ""
```

Comment:
I'm not sure if it's possible to have multiple route bindings for the same route id and service instance id combo

Reply:
I get this error when I tried it:
{"errors":[{"detail":"The route and service instance are already bound.","title":"CF-ServiceInstanceAlreadyBoundToSameRoute","code":130008}]}